### DISCUSSION TITLE
How can I get a diff object of a triple dot diff?

### DISCUSSION THREAD

**Discussion by itamarsch** [gitpython-developers/GitPython#1851] [Q&A]
I'm trying to compare the diffs between two branches for a git forge project and I tried using:

```python
diff = repo.git.diff(f"{branch1}...{branch2}")
```

But the diff result is a string and I need a diff object.
Is there a way to create a diff object from this string?

**Comment by Byron** [comment]
This just invokes the `git` command, without further processing.

In order to get diffs, the [higher-level API](https://gitpython.readthedocs.io/en/latest/quickstart.html#diffs) is needed, but I am not sure if diffs can be specified using the `a...b` syntax. It's certainly worth to just try it in place where the documentation speaks about refs or revs.
